snscrape/modules/telegram.py

```diff
@@ -161,48 +161,23 @@ def _soup_to_items(self, soup, pageUrl, onlyUsername = False):
 						# resp = self._get(image[0])
 						# encoded_string = base64.b64encode(resp.content)
 					# Individual photo or video link
-					continue
```

Comment:
I had to remove all of these `continue`s in order to get it to behave -- I think that the video or voice messages must've matched one of the earlier cases.  Not sure if you think there's a better solution, @loganwilliams , or if the performance hit is bad enough that I've gotta figure one out.